connector/src/main/java/org/geysermc/connector/utils/StatisticsUtils.java

```diff
@@ -231,4 +205,31 @@ private static String getItemTranslateKey(String item, String language) {
         }
         return translatedItem;
     }
+
+    private static String translate(String keys, String locale) {
```

Comment:
I feel like using regex and all this is overkill. It feels like you really wanted to make the form code clean at the expense of... this. At least just make this a function that takes in the locale and then two strings and return the same thing we were doing before. :P